packages/ui/react/src/icons/iconLoader.tsx

```diff
@@ -192,6 +193,7 @@ const iconList = [
   { type: 'icon', name: 'unlink', component: BiUnlink },
   { type: 'icon', name: 'user', component: MdPersonOutline },
   { type: 'icon', name: 'warning', component: MdWarning },
+  { type: 'icon', name: 'github', component: RiGithubFill },
```

Comment:
Why another `GitHub` icon is being added here?
* One already exists at https://github.com/devopness/devopness/pull/1945/files#diff-de3ffd4262a71dc04f3d3d7652145d9424ccf3a976b1f92d1213195fd555f425R214